packages/wrangler/package.json

```diff
@@ -84,7 +84,7 @@
 		"resolve.exports": "^2.0.2",
 		"selfsigned": "^2.0.1",
 		"source-map": "^0.6.1",
-		"unenv": "npm:unenv-nightly@2.0.0-1724863496.70db6f1",
+		"unenv": "npm:unenv-nightly@2.0.0-1725621861.f1c0bb8",
```

Comment:
There is actually a new release just out 30 mins ago: https://www.npmjs.com/package/unenv-nightly/v/2.0.0-1725898392.f1c0bb8

Is it worth bumping to that?

Reply:
Fixed in the active PR
https://github.com/cloudflare/workers-sdk/pull/6654#discussion_r1750594511